suggestedimages/locales/list_of_wiktionaries.py

```diff
@@ -1,171 +1,176 @@
-    'bg': 'bg', # Bulgarian
-    'ur': 'ur', # Urdu
-    'cy': 'cy', # Welsh
-    'ps': 'ps', # Pashto
-    'tg': 'tg', # Tajik
-    'kbd': 'kbd', # Kabardian
-    'he': 'he', # Hebrew
-    'vo': 'vo', # Volapük
-    'om': 'om', # Oromo
-    'sl': 'sl', # Slovenian
-    'af': 'af', # Afrikaans
-    'nan': 'nan', # Min Nan Chinese
-    'ms': 'ms', # Malay
-    'scn': 'scn', # Sicilian
-    'tl': 'tl', # Tagalog
-    'pa': 'pa', # Punjabi
-    'fy': 'fy', # Western Frisian
-    'sw': 'sw', # Swahili
-    'kk': 'kk', # Kazakh
-    'ka': 'ka', # Georgian
-    'nn': 'nn', # Norwegian Nynorsk
-    'min': 'min', # Minangkabau
-    'lv': 'lv', # Latvian
-    'nds': 'nds', # Low German
-    'gor': 'gor', # Gorontalo
-    'sq': 'sq', # Albanian
-    'lb': 'lb', # Luxembourgish
-    'bs': 'bs', # Bosnian
-    'co': 'co', # Corsican
-    'mn': 'mn', # Mongolian
-    'pnb': 'pnb', # Western Punjabi
-    'nah': 'nah', # Nāhuatl
-    'yue': 'yue', # Cantonese
-    'ckb': 'ckb', # Central Kurdish
-    'sa': 'sa', # Sanskrit
-    'diq': 'diq', # Zazaki
-    'km': 'km', # Khmer
-    'be': 'be', # Belarusian
-    'vec': 'vec', # Venetian
-    'nia': 'nia', # Nias
-    'tk': 'tk', # Turkmen
-    'mk': 'mk', # Macedonian
-    'sm': 'sm', # Samoan
-    'hsb': 'hsb', # Upper Sorbian
-    'ks': 'ks', # Kashmiri
-    'shy': 'shy', # Shawiya
-    'bcl': 'bcl', # Central Bikol
-    'su': 'su', # Sundanese
-    'ga': 'ga', # Irish
-    'btm': 'btm', # Batak Mandailing
-    'gd': 'gd', # Scottish Gaelic
-    'an': 'an', # Aragonese
-    'gom': 'gom', # Goan Konkani
-    'mr': 'mr', # Marathi
-    'ha': 'ha', # Hausa
-    'wo': 'wo', # Wolof
-    'mni': 'mni', # Manipuri
-    'ia': 'ia', # Interlingua
-    'bjn': 'bjn', # Banjar
-    'ang': 'ang', # Old English
-    'mt': 'mt', # Maltese
-    'tt': 'tt', # Tatar
-    'sd': 'sd', # Sindhi
-    'blk': 'blk', # Pa'O
-    'fo': 'fo', # Faroese
-    'so': 'so', # Somali
-    'si': 'si', # Sinhala
-    'gn': 'gn', # Guarani
-    'ie': 'ie', # Interlingue
-    'mi': 'mi', # Māori
-    'csb': 'csb', # Kashubian
-    'ug': 'ug', # Uyghur
-    'guw': 'guw', # Gun
-    'st': 'st', # Southern Sotho
-    'hif': 'hif', # Fiji Hindi
-    'jbo': 'jbo', # Lojban
-    'roa-rup': 'rup', # Aromanian
-    'kl': 'kl', # Kalaallisut
-    'zu': 'zu', # Zulu
-    'ay': 'ay', # Aymara
-    'ln': 'ln', # Lingala
-    'yi': 'yi', # Yiddish
-    'gu': 'gu', # Gujarati
-    'kcg': 'kcg', # Tyap
-    'na': 'na', # Nauru
-    'gv': 'gv', # Manx
-    'kw': 'kw', # Cornish
-    'tpi': 'tpi', # Tok Pisin
-    'am': 'am', # Amharic
-    'ne': 'ne', # Nepali
-    'rw': 'rw', # Kinyarwanda
-    'ts': 'ts', # Tsonga
-    'ig': 'ig', # Igbo
-    'qu': 'qu', # Quechua
-    'ss': 'ss', # Swati
-    'iu': 'iu', # Inuktitut
-    'chr': 'chr', # Cherokee
-    'dv': 'dv', # Divehi
-    'ti': 'ti', # Tigrinya
-    'tn': 'tn', # Tswana
+from collections import namedtuple
+
+
+Wikt = namedtuple("Wikt", "subdomain language_code local_name english_name")
```

Comment:
I thought the name isn't really important here, because it's just used here to give column names for the table with the help of a named tuple. I think a longer name would be distracting here.

Reply:
Would this be better?

```python
W = namedtuple("WiktionaryInfo", "subdomain language_code local_name english_name")

wiktionary_info = {
    "af":	W("af",	"af",	"Afrikaans",	"Afrikaans"),
    "am":	W("am",	"am",	"አማርኛ",	"Amharic"),
   ...
```